app/assets/javascripts/modules/moj.sentry.js

```diff
@@ -7,8 +7,12 @@
       this.raven = Raven;
       this.sentry_js_dsn = $(this.el).data('sentry-js-dsn');
       this.raven.config(this.sentry_js_dsn).install();
-      var sentry_module = this;
 
+      // Capture any uncaught errors
+      // window.onerror = $.proxy(this.handleException, this);
```

Comment:
yes, i'm not convinced this is needed and it seems to obliterate the where the actual error is